### ISSUE TITLE
Request - Config Save

### ISSUE THREAD

**Issue by rdf123** [boxeehacks/boxeehack#353]
Hi,

It would be great if you could save your directories/media locations etc on a usb key and just import it instead of having to manually input all the info every time we do a reset/clear media database.  Would that be possible?

Thank you for all the hard work.

**Comment by ardenee** [comment]
Yup, this would be handy, and save in plain text for easy editing should their be a need!

An additional save/restore option added to boxeeboxip:8080 would be handy!

**Comment by boxeehacks** [comment]
Good suggestion. Will look into this...
